### ISSUE TITLE
[Export-M365DSCConfiguration] Tenant-wide export fails.

### ISSUE THREAD

**Issue by SebastianGoers** [microsoft/Microsoft365DSC#954]
#### Details of the scenario you tried and the problem that is occurring
An export of a tenant configuration fails. The script is prompting the user for input (requiring a few parameters). I used Export-M365DSCConfiguration to start the export.
#### Verbose logs showing the problem
```
PS C:\Windows\system32> Export-M365DSCConfiguration -Quiet
[1/112] Extracting [AADApplication]...
    |---[1/1] [****]Teams-****-Automation✅
[2/112] Extracting [AADGroupsNamingPolicy]...✅
[3/112] Extracting [AADGroupsSettings]...✅
[4/112] Extracting [AADMSGroupLifecyclePolicy]...✅
[5/112] Extracting [AADPolicy]...
    |---[1/1] B2BManagementPolicy✅
[6/112] Extracting [AADRoleDefinition]...
    |---[1/70] Company Administrator
cmdlet Get-TargetResource at command pipeline position 1
Supply values for the following parameters:
IsEnabled:
```

#### Suggested solution to the issue
None, yet.
#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
N/A Export only.
```

**Comment by jeff29922** [comment]
@SebastianGoers I am going to grab this one as well.  I am testing in the lab with the newest software.  I'll update you ASAP.

**Comment by jeff29922** [comment]
I reached out to @SebastianGoers via the team site for an update.  This issue has bee resolved.  Sebastian can you please elaborate on the fix

**Comment by NikCharlebois** [comment]
Closing. This issue was fixed in 1.20.1209.1
